faiss/gpu/StandardGpuResources.cpp

```diff
@@ -375,6 +394,19 @@ cudaStream_t StandardGpuResourcesImpl::getDefaultStream(int device) {
     return defaultStreams_[device];
 }
 
+raft::handle_t& StandardGpuResourcesImpl::getRaftHandle(int device) {
+    initializeForDevice(device);
+
+    auto it = raftHandles_.find(device);
```

Comment:
`initializeForDevice` already constructs something in `raftHandles_` for this device, so we can simply `return raftHandles_[device];` like below instead of this?